packages/contracts-bedrock/src/Safe/GetSigners.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+library GetSigners {
+    /// @notice Splits signature bytes into `uint8 v, bytes32 r, bytes32 s`.
+    ///         Copied directly from
+    /// https://github.com/safe-global/safe-contracts/blob/e870f514ad34cd9654c72174d6d4a839e3c6639f/contracts/common/SignatureDecoder.sol
+    /// @dev Make sure to perform a bounds check for @param pos, to avoid out of bounds access on @param signatures
```

Comment:
Just curious on this natspec format, does `@param pos` link to that param in `forge doc`?

Reply:
No idea, it's straight outta the safe repo: https://github.com/safe-global/safe-contracts/blob/e870f514ad34cd9654c72174d6d4a839e3c6639f/contracts/common/SignatureDecoder.sol#L11